### DISCUSSION TITLE
can I sync bookmarks and tabs at same time?

### DISCUSSION THREAD

**Discussion by dkebler** [floccusaddon/floccus#1394] [Q&A] [answered]
I want to sync my tabs and also my "bookmarks bar" bookmarks subfolder.

In the setup it's a radio button so can only choose sync tabs or sync booksmarks.  Does that mean they both can't be done together?   

If so can I have to set up a second account (on both machines) one pair for booksmarks and one pair for tabs?   Just a bit confused

**Comment by marcelklehr** [comment, marked as answer]
You can sync one thing per account. If you want to sync bookmarks and tabs, you need to accounts. You should not sync them both to the same target, though, because otherwise it will open all your bookmarks as tabs ;)
